### PR TITLE
added recipe for `pyppmd`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/python this is ready for review!

**Comment by sarthakpati** [comment]
@conda-forge/python this is ready for review!

**Comment by sarthakpati** [comment]
Glad for the help, @dopplershift!

BTW the line `src_root.joinpath(s).as_posix()` works perfectly fine on Windows! 😄

**Comment by dopplershift** [comment]
If you look carefully at the Windows failures before, it's not from the actual process of building the extension module, but when processing a `SOURCES.txt` file--it turns out the source distribution has a stray `.egg-info` directory include with some bad paths (including the one complained about).

File: recipes/pyppmd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools >=45.0
+    - setuptools-scm >=6.0.1
+    - importlib_metadata
```
Comment (dopplershift) [inline recipes/pyppmd/meta.yaml:30]: `importlib_metadata` doesn't look like it should be needed at build-time.

Comment (sarthakpati) [reply recipes/pyppmd/meta.yaml]: Done!

**Comment by sarthakpati** [comment]
> If you look carefully at the Windows failures before, it's not from the actual process of building the extension module, but when processing a `SOURCES.txt` file--it turns out the source distribution has a stray `.egg-info` directory include with some bad paths (including the one complained about).

Ah, got it! Thank you so much!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**